Note to Clarity Connection Differences (#2350)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/meeting-policies-in-teams.md

- [Allow dial-in users to bypass the lobby](#allow-dial-in-users-to-bypass-the-lobby-coming-soon)
- [Allow organizers to override lobby settings](#allow-organizers-to-override-lobby-settings-coming-soon)

> [!NOTE]

> > Joining a meeting will vary depending on the settings for each Teams group and the method of connecting. If your group has audio conferencing and you are connecting through audio conferencing, then refer to [Audio Conferencing in Office 365](https://docs.microsoft.com/microsoftteams/audio-conferencing-in-office-365). If your Teams group does not have audio conferencing, then refer to [Join a meeting in Teams](https://support.office.com/article/join-a-meeting-in-teams-1613bb53-f3fa-431e-85a9-d6a91e3468c9).

Reviewer: ```suggestion
>Options to join a meeting will vary, depending on the settings for each Teams group, and the connection method. If your group has audio conferencing, and uses it to connect, see [Audio Conferencing in Office 365](https://docs.microsoft.com/microsoftteams/audio-conferencing-in-office-365). If your Teams group does not have audio conferencing, refer to [Join a meeting in Teams](https://support.office.com/article/join-a-meeting-in-teams-1613bb53-f3fa-431e-85a9-d6a91e3468c9).
```

> 

Reviewer: ```suggestion
### Automatically admit people
```
